src/bidiMapper/domains/context/PreloadScriptStorage.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import crypto from 'crypto';
```

Comment:
I think we need to make a wrapper so that we can use choose the right crypto module in the browser and in Node. It looks like this would only work in Node.

Reply:
I'd reconsider importance of having a proper crypto-resistant UUID's for intended purposes. For providing startup script IDs it doesn't really matter. For the magical DOM property for channel it's debatable.

Reply:
Do you have an example of how I can use a wrapper for this?

Reply:
Ooops https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID
> Secure context: This feature is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS), in some or all [supporting browsers](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility).

Reply:
`self.crypto.randomUUID()` should be cross-env as long as the origin is secure.

Reply:
the origin is `about:blank`

Reply:
As discussed elsewhere, we will proceed with `uuid` here (to unblock), and then (soon) refactor it with a simple polyfill.

Reply:
go/webdriver:bidi-import-crypto-module